packages/serverless/package.json

```diff
@@ -50,6 +50,7 @@
   "dependencies": {
     "@apollo/server": "4.11.0",
     "@orochi-network/framework": "^2.3.19",
+    "@orochi-network/queue": "^0.1.11",
```

Comment:
Are we need it here?

Reply:
Yes, I use the Fill function to perform concurrency task

Reply:
Removed, We don't use Fill anymore